integration/oidc/authorization-code/README.md

```diff
@@ -0,0 +1,197 @@
+
+### Create the application registration
+```powershell
+# Replace `WorkflowGen GraphQL API` with the name of the registered WorkflowGen
+# GraphQL API application including the backticks if you named it something else.
+$webApiApp = & az ad app list `
+    --query '[?displayName == `WorkflowGen GraphQL API`] | [0]' `
+    | ConvertFrom-Json `
+    | Select-Object -First 1
+$password = [guid]::NewGuid().ToString()
+$webAppUrl = "http://localhost:8080"
+
+# Create the application registration.
+$webAppClient = & az ad app create `
+    --display-name "My aspnet core Web App" `
+    --homepage $webAppUrl `
+    --identifier-uris $webAppUrl `
+    --password $password `
+    --end-date '2299-12-31T11:59:59+00:00' `
+    --reply-urls "$webAppUrl/signin" `
+    | ConvertFrom-Json
+
+# Create a service principal for the registration
+& az ad sp create --id $webAppClient.appId
+
+# Add required permissions for the GraphQL API access
+& az ad app permission add `
+    --id $webAppClient.appId `
+    --api $webApiApp.appId `
+    --api-permissions "$($webApiApp.oauth2Permissions.id)=Scope"
+
+# Grant the required permissions for the GraphQL API Access
+& az ad app permission grant `
+    --api $webApiApp.appId `
+    --id $webAppClient.appId
+```
+
+You will need the following information for the example configuration:
+
+* The client id. This is the value of `$webAppclient.appId`.
+* The client secret. This is the value of `$password`.
+* The GraphQL API App Id URI. This is the value of `$webApiApp.identifierUris[0]`.
+* The Azure AD Directory ID. Run the following command to get it:
+
+    ```powershell
+    az account show | ConvertFrom-Json | ForEach-Object tenantId
+    ```
+
+## Create a New App Registration in Azure AD (Azure Web Portal)
+To represent this example application that wants to access WorkflowGen's
+GraphQL API, we need to register it to Azure AD and give it access to the API.
+This way, when requesting the GraphQL API resource in the OIDC protocol, you
+will get an access token that is a valid JWT and that can be transmitted to
+the API for authorization.
+
+Keep in mind that the user that will authenticate with this application must
+exists in WorkflowGen.
+
+1. In Azure AD, go to the `App Registrations` section.
+1. Click on the `New application registration` button.
+1. Enter the following information:
+
+    Name: `My aspnet core Web App`
+    Application Type: `Web app / API`
+    Sign-on URL: `http://localhost:8080`
+
+1. When you are ready, click on the `Create` button.
+
+You now have registered this example application. You now need to finish
+configuring it in order to get the auhtentication working properly.
+
+1. Go to the `My aspnet core Web App` application registration page in Azure AD.
+1. Copy the Application ID and save it for later use. This is the `Client Id`
+of the application.
+1. Click on the `Settings` button.
+1. Go to the `Reply URLs` section.
```

Comment:
1. In the **Reply URLs** section, add `http://localhost:8080/signin` to the list, then click **Save**.